simcon_project/users/views/student_home.py

```diff
@@ -5,23 +5,27 @@
 
 
 class StudentHomeTable(tables.Table):
-    name = tables.Column(linkify={"viewname": "TemplateStartView", "args": [tables.A("conversation_templates__name")]},
+    name = tables.Column(linkify=("conversation-start",
+                                  {"ct_id": tables.A("conversation_templates__id"),
+                                   "assign_id": tables.A("id")
+                                   }),
                          accessor='conversation_templates__name',
                          verbose_name='Template Name')
     date_assigned = tables.Column(verbose_name='Date Assigned')
     completion_date = tables.Column(accessor='conversation_templates__template_responses__completion_date',
                                     verbose_name='Last Response')
 
 
-@login_required
-def StudentView(request):
+@login_required(login_url="/accounts/login/")
```

Comment:
whats the differnce between that and just login required?

Reply:
Commented on your PR addressing this